support/helpers/temporary_workbook.rb

```diff
@@ -1,12 +1,12 @@
 class TemporaryWorkbook
-  def initialize headings, data
+  def initialize headings, data, blacklist = []
     dir = Dir.mktmpdir
     @file = File.new("#{dir}/foo.xlsx", 'w+')
-    workbook = Support::Factories::Spreadsheet.build(headings, data)
+    workbook = Support::Factories::Spreadsheet.build(headings, data, blacklist)
```

Comment:
Could we keep this generic by creating introducing a further encapsulation for Worksheet?

For example,

```ruby
metadata_headings = blacklist[0]
metadata_data = blacklist[1..-1]
mediators_worksheet = Support::Factories::Worksheet.build(headings, data)
metadata_worksheet = Support::Factories::Worksheet.build(metadata_headings, metadata_data)
workbook = Support::Factories::Spreadsheet.build([mediators_worksheet, metadata_worksheet])
```